packages/echo-core/src/services/authentication/authProviderConfig.ts

```diff
@@ -1,5 +1,4 @@
-import { AccountInfo, EndSessionRequest, SilentRequest } from '@azure/msal-browser';
-import { RedirectRequest } from '@azure/msal-browser/dist/src/request/RedirectRequest';
+import { AccountInfo, EndSessionRequest, RedirectRequest, SilentRequest } from '@azure/msal-browser';
```

Comment:
Lines 1 and 2 were merged into this one line, because in the new version of `@azure/msal-browser` on the previous path there was no `RedirectRequest`.

Just importing it now from the root (as it's exported there)